tools/dependency-check.sh

```diff
@@ -47,6 +47,15 @@ check_shasum() {
 
 check_cmake() {
   check_binary_present "cmake"
+
+  CMAKE_FULL_VERSION=$(cmake --version | head -n1 | cut -d " " -f 3)
+  CMAKE_MAJOR_VERSION=$(echo "${CMAKE_FULL_VERSION}" | cut -d "." -f 1)
+
+  if (( "${CMAKE_MAJOR_VERSION}" != "3" )); then
+    echo >&2 "Pyodide requires cmake 3."
+    echo >&2 "If you are on macOS brew would install higher version. Activating conda will fix it"
```

Comment:
I do not want to recommend conda, unless there is no other easy way to install the old version of cmake in MacOS. I think the way that @youknowone added, installing the CMake Python package, which bundles the CMake binary in it, was a reasonable alternative, as we need Python installed anyway.

Reply:
Additionally, [this](https://github.com/pyodide/pyodide/blob/ab45a2025559b18dd2cbd896baccc00660b97ea5/environment.yml#L21-L22) seems to be intended for situations like this, so I thought it would make sense to leverage the existing managed environment.

Reply:
@ryanking13 shared his opinion in another channel, and now I understand that pip is more approachable than conda for most people.

Applied this!
d7f4352a441cf429f7f2512ad1417676d3cd34ef

If you want to squash these commits, I will do it.